chat/src/main/java/com/travelbuddy/chat/service/MongoService.java

```diff
@@ -0,0 +1,89 @@
+package com.travelbuddy.chat.service;
+
+import com.travelbuddy.chat.entity.ChatRoom;
+import com.travelbuddy.chat.entity.OnlineUsers;
+import com.travelbuddy.chat.exception.RoomNotFoundException;
+import com.travelbuddy.chat.model.Message;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import org.springframework.data.mongodb.core.query.Criteria;
+import org.springframework.data.mongodb.core.query.Query;
+import org.springframework.data.mongodb.core.query.Update;
+import org.springframework.stereotype.Service;
+
+import java.util.*;

```

Comment:
Try and avoid Wildcards

Reply:
Oh Sure , not sure how it came , will fix that in next commit